docs/book/src/reference/labels_and_annotations.md

```diff
@@ -1,62 +1,62 @@
+| machine.cluster.x-k8s.io/exclude-node-draining                   | It explicitly skips node draining if set.    | User | Machines |
+| machine.cluster.x-k8s.io/exclude-wait-for-node-volume-detach     | It explicitly skips the waiting for node volume detaching if set.      | User | Machines |
+| machinedeployment.clusters.x-k8s.io/desired-replicas             | It is the desired replicas for a machine deployment recorded as an annotation in its machine sets. Helps in separating scaling events from the rollout process and for determining if the new machine set for a deployment is really saturated.   |    Cluster API | Machine deployment's machine sets |
+| machinedeployment.clusters.x-k8s.io/max-replicas                 | It is the maximum replicas a deployment can have at a given point, which is machinedeployment.spec.replicas + maxSurge. Used by the underlying machine sets to estimate their proportions in case the deployment has surge replicas.  |     Cluster API    | Machine deployment's machine sets |
+| machinedeployment.clusters.x-k8s.io/revision                     | It is the revision annotation of a machine deployment's machine sets which records its rollout sequence.         |         |              |
+| machinedeployment.clusters.x-k8s.io/revision-history             | It maintains the history of all old revisions that a machine set has served for a machine deployment.   | Cluster API | Machine deployment's machine sets |
+| pre-drain.delete.hook.machine.cluster.x-k8s.io                   | It specifies the prefix we search each annotation for during the pre-drain.delete lifecycle hook to pause reconciliation of deletion. These hooks will prevent removal of draining the associated node until all are removed.   | User | Machines |
+| pre-terminate.delete.hook.machine.cluster.x-k8s.io               | It specifies the prefix we search each annotation for during the pre-terminate.delete lifecycle hook to pause reconciliation of deletion. These hooks will prevent removal of an instance from an infrastructure provider until all are removed.   | User | Machines |
+| topology.cluster.x-k8s.io/defer-upgrade                          | It can be used to defer the Kubernetes upgrade of a single MachineDeployment topology. If the annotation is set on a MachineDeployment topology in Cluster.spec.topology.workers, the Kubernetes upgrade for this MachineDeployment topology is deferred. It doesn't affect other MachineDeployment topologies.                                                                                               | Cluster API |	MachineDeployment topology |
```

Comment:
```suggestion
| topology.cluster.x-k8s.io/defer-upgrade                          | It can be used to defer the Kubernetes upgrade of a single MachineDeployment topology. If the annotation is set on a MachineDeployment topology in Cluster.spec.topology.workers, the Kubernetes upgrade for this MachineDeployment topology is deferred. It doesn't affect other MachineDeployment topologies.                                                                                               | Cluster API |	MachineDeployments in Cluster.topology |
```